apps/teamsjs-diagnostic-app/src/apis/GeolocationApi.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
```

Comment:
Why did you omit the permission functions from geolocation?

Reply:
I'll add them in!